### ISSUE TITLE
SDK meeting APIView link in Github issue is wrong when no link was provided

### ISSUE THREAD

**Issue by maririos** [Azure/azure-sdk-tools#4414]
When a user creates a Follow-up meeting, they are asked to select the language. They do not have to provide the APIView link. When this happens, the issue that is created shows the language as a hyperlink. This link is set to the GH issue. This could be confusing.
Proposal is to leave it as plain text if no link is provided (can we do this?)

**Comment by maririos** [comment]
>Proposal is to leave it as plain text if no link is provided (can we do this?)

This won't work because of how our template works.

Another option could be to include a new field with the Languages and then having a separate one for APIView links. always populate the languages one, and only include the APIView links when present

**Comment by maririos** [comment]
This is creating more confusion and causing meetings to be rescheduled. Bumping priority

**Comment by maririos** [comment]
Example: https://github.com/Azure/azure-sdk/issues/6125

**Comment by JonathanCrd** [comment]
>> Proposal is to leave it as plain text if no link is provided (can we do this?)
>
>This won't work because of how our template works.

Actually, this can work. I was able to change the parameter from the app to send a plain text instead of an email to the template, and it worked. See https://github.com/azure-sdk/TestingForEngagementExperience-pr/issues/113 for reference. With this approach, neither the email templates nor the app functionality to add API View links will be changed.

To make it more obvious, I attached `(pending)` next to the language. @maririos What do you think about this?

Additionally, I think is valuable to add more warnings or messages to let the user know that we expect the API View Links there before the meeting happens. There is no clear message for this when a user adds a language without a link.

**Comment by JonathanCrd** [comment]
If we want to make it more explicit, I could add a new field called **Targeted languages** or similar. But with this approach, the Email Template must be changed, same for the App's API View Links feature, and UI too.

**Comment by maririos** [comment]
> To make it more obvious, I attached (pending) next to the language. @maririos What do you think about this?

We could start just with the plain text as that was the recommended requirement from @ronniegeraghty and later on, if feedback tells us to add `(pending)` we can do that

> Additionally, I think is valuable to add more warnings or messages to let the user know that we expect the API View Links there before the meeting happens. There is no clear message for this when a user adds a language without a link.

Sure! And also, we need to improve message that when they update the APIView link in the Scheduler, they still need to hit the `Update` button for the meeting so the data can be propadated everywhere

**Comment by maririos** [comment]
> If we want to make it more explicit, I could add a new field called Targeted languages or similar. But with this approach, the Email Template must be changed, same for the App's API View Links feature, and UI too.

We can make the email template match the GH one. There is this issue https://github.com/Azure/azure-sdk-tools/issues/5021

**Comment by JonathanCrd** [comment]
I enhanced the UI to include more descriptive messages to make clear that API View Links are required for Follow Ups reviews.

> And also, we need to improve message that when they update the APIView link in the Scheduler, they still need to hit the Update button for the meeting so the data can be propadated everywhere

To avoid confusion, I disabled the success notification and added a local loading instead. Now only the Domains table loads when someone adds or removes a language. However, in the future we should patch the tables inside the Create/Update button.

And last but not least, GitHub issues now will display only the text for the domain if no link was provided.
